addons/che/templates/che-server-template.yaml

```diff
@@ -96,8 +96,8 @@ objects:
             value: "openshift"
           - name: CHE_INFRA_KUBERNETES_BOOTSTRAPPER_BINARY__URL
             value: "${PROTOCOL}://che-${NAMESPACE}.${ROUTING_SUFFIX}/agent-binaries/linux_amd64/bootstrapper/bootstrapper"
-          - name: CHE_INFRA_KUBERNETES_MACHINE__START__TIMEOUT__MIN
-            value: "5"
+          - name: CHE_INFRA_KUBERNETES_WORKSPACE__START__TIMEOUT__MIN
+            value: "10"
```

Comment:
What does this time out do?

Reply:
AFAIU it defines maximal time for workspace to be running. If for any reason the workspace is not ready in that time the start is stopped and the startup is reported as failed. Default value in Che is 7 minutes.

Reply:
They changed the naming some time ago :rofl: https://github.com/eclipse/che/commit/806a6da184c069e8cc0f6953c7af97e82994132a